satpy/composites/__init__.py

```diff
@@ -631,18 +631,14 @@ def __call__(self, projectables, **kwargs):
         try:
             coszen = np.cos(np.deg2rad(projectables[2 if self.day_night == "day_night" else 1]))
         except IndexError:
-            from pyorbital.astronomy import cos_zen
+            from satpy.modifiers.angles import get_cos_sza
```

Comment:
This can be moved to the top of the module.

Reply:
I actually had it at the top originally, but got a 'circular import' error. Will try again later.

Reply:
Ah ok. Then leave it. The modifier in the `satpy/modifiers` are loading the base compositor from this `composites/__init__.py`. Yet another reason this module needs to be broken up.